spec/index.bs

```diff
@@ -4784,10 +4862,11 @@ dictionary GPUFragmentState: GPUProgrammableStage {
 </script>
 
 <div algorithm>
-    <dfn abstract-op>validating GPUFragmentState</dfn>(|descriptor|)
+    <dfn abstract-op>validating GPUFragmentState</dfn>(|descriptor|, |device|)
         Return `true` if all of the following requirements are met:
 
-        - |descriptor|.{{GPUFragmentState/targets}}.length must be &le; 4.
+        - |descriptor|.{{GPUFragmentState/targets}}.length must be &le;
+            |device|.limits.{{supported limits/maxColorAttachments}}.
```

Comment:
can we actually write `|device|.limits`? I thought we have to write `|device|.{{GPUDevice/[[limits]]}}` instead

Reply:
I think either `|device|.{{GPUDevice/limits}}` or `|device|.{{GPUDevice/[[device]]}}.{{device/[[limits]]}}`, but the former seems fine to me.